lib/src/telegram/models/audio.dart

```diff
@@ -52,4 +52,18 @@ class Audio {
 
   factory Audio.fromJson(Map<String, dynamic> json) => _$AudioFromJson(json);
   Map<String, dynamic> toJson() => _$AudioToJson(this);
+
+  @override
+  String toString() => '''Audio
```

Comment:
what are these `toString` methods for?

Reply:
These are methods to debug that I use internally on my fork